src/main/webapp/app/course/manage/overview/course-management-card.scss

```
@@ -0,0 +1,94 @@
+@import '../../../overview/_overview-vars.scss';
+
+.card {
+    min-height: 100%;
+    max-width: 2200px;
+    .card-heading {
```

```suggestion
    
    .card-heading {
```
Codacy: Expected empty line before rule (rule-empty-line-before)